OmniBLE/PumpManager/OmniBLEPumpManager.swift

```diff
@@ -1705,8 +1705,11 @@ extension OmniBLEPumpManager: PumpManager {
                 return
             }
 
+            // Use a maximum programReminderInterval value of 0x3F to denote an automatic bolus in the communication log
+            let programReminderInterval: TimeInterval = automatic ? TimeInterval(minutes: 0x3F) : 0
+
             let beep = self.confirmationBeeps
-            let result = session.bolus(units: enactUnits, acknowledgementBeep: beep, completionBeep: beep)
+            let result = session.bolus(units: enactUnits, acknowledgementBeep: beep, completionBeep: beep, programReminderInterval: programReminderInterval)
```

Comment:
A better way of doing this would be to log extra metadata in the device logs themselves. Shortcuts like this end up making the code harder to read (why are we sending a reminder for a bolus?) and make later changes harder (what about tracking other metadata about the bolus?)

Reply:
Having this information in the message logs has been helpful to people providing Omnipod support across multiple DIY implementations. This has been done like this ever since the OmniKit enactBolus has had a parameter to indicate if a bolus is automatic (in private implementations for over 2 years now as well as in has been Loop dev, FreeAPS and FreeAPSX for quite some time already). Loop's OmniBLE shouldn't be different in this regard than OmniKit on Loop/FreeAPS/FreeAPSX as well as OmniBLE on FreeAPS and eventually FreeAPSX. The only reason why this isn't already here was simply because OmniBLE was forked for Loop dev before the post v2.2.6 OmniKit changes were applied.